pkg/cvo/cvo.go

```diff
@@ -139,6 +139,10 @@ type Operator struct {
 	// lastAtLock guards access to controller memory about the sync loop
 	lastAtLock          sync.Mutex
 	lastResourceVersion int64
+
+	// excludeManifestsFilter is a set of regular expresions that are used
+	// to exclude manifest file names that match them.
+	excludeManifestsFilter payload.RegexpFilter
```

Comment:
I think the payload should be required to identify the manifests that match a string, rather than a regex.  I do not like regex and I don't want arbitrary regular expressions.

Reply:
ack, string matching should be sufficient for what we need, will switch to that

Reply:
> I think the payload should be required to identify the manifests that match a string...

I'm trying to catch up with the previous discussion in this PR, but if the folks setting this are creating release images, can't they just remove the manifests they don't want pushed?  And if they aren't writing to release images, why is [the existing `overrides`][1] insufficient.  Do we have [the enhancement][2] explaining how this is expected to fit together yet?

[1]: https://github.com/openshift/api/blob/a27ff30ebf098b145a702153b2a466a0f9e6e794/config/v1/types_cluster_version.go#L68-L72
[2]: https://github.com/openshift/cluster-version-operator/pull/252#issuecomment-549687836

Reply:
We don't want a separate release image for this case. The release image should be the same we use everywhere.

And no, the enhancement is not ready, since I think the design of profiles needs a little bit more discussion. The purpose of this PR is simply to a stop-gap in 4.3 until we get to profiles.